src/arm/linux/aarch32-isa.c

```diff
@@ -147,6 +147,8 @@ void cpuinfo_arm_linux_decode_isa_from_proc_cpuinfo(
 				"VDOT instructions disabled: cause occasional SIGILL on Spreadtrum SC9863A");
 		} else if (chipset->series == cpuinfo_arm_chipset_series_unisoc_t && chipset->model == 310) {
 			cpuinfo_log_warning("VDOT instructions disabled: cause occasional SIGILL on Unisoc T310");
+		} else if (chipset->series == cpuinfo_arm_chipset_series_unisoc_ums && chipset->model == 312) {
```

Comment:
we you able to repro this on armv8 kernel? And does this fix it? Ok with this change even if you couldn't TBH.

Reply:
Unfortunately. I don't have a device with UMS312 and armv8 kernel. From the crash logs, they were mostly (maybe all?) armv7 kernels, so it may be okay. If we see issues on armv8 kernels, I can take as a follow-up.